contracts/contracts/BadgerScout.sol

```diff
@@ -457,4 +415,44 @@ contract BadgerScout is IBadgerScout, Ownable, ERC1155 {
             _data
         );
     }
+
+    /*//////////////////////////////////////////////////////////////
+                        INTERNAL VALIDATORS
+    //////////////////////////////////////////////////////////////*/
+
+    /**
+     * @notice Enforces that the length of three arrays are equal.
+     * @param _length1 The first array length.
+     * @param _length2 The second array length.
+     * @param _length3 The third array length.
+     */
+    function _validateLengths(uint256 _length1, uint256 _length2, uint256 _length3) internal pure {
+        require(
+            _length1 == _length2 && _length2 == _length3,
+            "BadgerScout::_validateLengths: Array lengths must be equal."
+        );
+    }
+
+    /**
+     * @notice Enforces that the length of two arrays are equal.
+     * @param _length1 The first array length.
+     * @param _length2 The second array length.
+     */
+    function _validateLengths(uint256 _length1, uint256 _length2) internal pure {
```

Comment:
Can this be implemented as `_isValidLengths(a,b) returns (bool)` so that the revert statements can have function-localized naming?

Prefer increased cost to decreased user experience as if they were to chain/multicall it would be almost impossible to determine the source of error without stack traces (which aren't a thing most solidity devs are adept enough to use.)

Reply:
Resolved by 9d907718e9f5f818ca451549e6d5fb6d9515af98